lang/go/go.talon

```diff
@@ -66,25 +113,46 @@ else <user.text> [over]:
     key("enter")
     insert(user.formatted_text(text, "PRIVATE_CAMEL_CASE"))
 
-state while: "while "
+[state] if (err|error):
+    insert("if err != nil {")
+    key("enter")
+
+[state] if not (err|error):
+    insert("if err == nil {")
+    key("enter")
```

Comment:
We should prob standardise on how all htese things with `{` followed by enter work, and do the same thing in all languages

Also please make "state" mandatory